stock_picking_report_product_sticker/models/stock_picking.py

```diff
@@ -20,7 +23,7 @@ class StockPicking(models.Model):
         store=False,
     )
 
-    @api.depends("picking_type_id.show_product_stickers")
+    @api.depends("picking_type_id")
     def _compute_show_product_stickers(self):
         for picking in self:
             picking.show_product_stickers = (
```

Comment:
This compute seems like it will fail now, as it assigns a bool value.

Reply:
Im using the same field, not a new one. The new type of field is Selection

Reply:
Ah yes, sorry indeed! 🤦🏼‍♂️